web-api/openapi/schemas/document.yml

```diff
@@ -1,15 +1,15 @@
 DocumentId:
-  description: Id of a document.
+  description:
+    $ref: './id.yml#/Id/description'
   $ref: './id.yml#/Id'
-  example: "document_id"
 
```

Comment:
in general syblings to items already defined in a `$ref` are not allowed and ignored in openapi. there are exceptions to `summary` and `description` which overwrite the `$ref`'s one.

hence i took the `$ref`ed description which is much more descriptive and also holds important information about constraints (we have to explicitly reference it here, otherwise references of references fail to resolve in the validator).

other items like `example` are ignored anyways, so i removed them. the generator will pick up the `example` of the reference.